commit ad5d0d1a0f938cfd4411bf3b08637e6a8c2f5c10 (refs/changes/74/4574/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:50:31+00:00 (1 year, 2 months ago)
    
    TRITON-612 fw-agent should be aware of alternate admin nictags (bump version)

diff --git a/package.json b/package.json
index 61ef70a..f8a2650 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "firewaller",
     "description": "SmartDataCenter Firewalling Agent",
-    "version": "1.4.0",
+    "version": "1.5.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
